pulp_ansible/app/urls.py

```diff
@@ -38,6 +38,11 @@
 ]
 
 v3_urls = [
+    path(
+        "metadata/",
+        views_v3.MutableCollectionMetadataViewset.as_view({"get": "list"}),
```

Comment:
![image](https://user-images.githubusercontent.com/17153022/95777451-94741880-0c9c-11eb-9562-c1b061af2361.png)

Reply:
Sync should check for the `last_updated`